packages/hdwallet-provider/package.json

```diff
@@ -10,28 +10,35 @@
   },
   "version": "1.0.18",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "prepare": "npx webpack --config webpack/webpack.config.js",
-    "test": "mocha --timeout 5000"
+    "build": "tsc",
+    "prepare": "yarn build",
+    "test": "yarn build && mocha --exit -r ts-node/register test/**/*.test.ts"
   },
   "dependencies": {
     "any-promise": "^1.3.0",
-    "bindings": "^1.3.1",
+    "bindings": "^1.5.0",
+    "bip39": "^2.4.2",
+    "debug": "^4.1.1",
+    "ethereum-protocol": "^1.0.1",
+    "ethereumjs-tx": "^2.1.1",
+    "ethereumjs-util": "^6.1.0",
+    "ethereumjs-wallet": "^0.6.3",
     "web3": "1.2.1",
-    "websocket": "^1.0.28"
+    "web3-provider-engine": "https://github.com/trufflesuite/provider-engine#web3-one",
+    "websocket": "^1.0.30"
```

Comment:
It seems we are not using it, can I remove it painlessly?

Reply:
Certainly!